lib/geokit-rails/acts_as_mappable.rb

```diff
@@ -141,12 +141,12 @@ def by_distance(options = {})
       end
 
       def closest(options = {})
-        by_distance(options).first(1)
+        by_distance(options).limit(1)
       end
       alias nearest closest
 
       def farthest(options = {})
-        by_distance(options).last(1)
+        by_distance(options).limit(1)
```

Comment:
@marcandre This won't work as you need to get the last, you'll need to add an order statement or something